WTWSMS/events/religious_events.txt

```diff
@@ -783,7 +801,7 @@ character_event = {
 			trait = cynical
 		}
 		modifier = {
-			factor = 0.25
+			factor = 0.1
```

Comment:
No. By lowering the percentage, I make it happen faster. The percentage was too high in that the Event MTTH wasn't lowered enough.